modules/ort/src/test/java/org/eclipse/sw360/antenna/ort/workflow/analyzers/OrtResultAnalyzerTest.java

```diff
@@ -89,19 +86,19 @@ public void testParseOrtDataWithoutAnalyzerResultsToArtifacts() throws URISyntax
                 .flatMap(s -> Stream.of(s.split("/n"))).toArray())
                 .hasSize(1+1); //+1 since first element of array will be empty due to parsing
 
-        assertThat(artifacts.stream().
-                map(artifact -> artifact.askForGet(ArtifactSourceUrl.class)).
-                filter(Optional::isPresent).
-                map(Optional::get).
-                filter(s -> !Objects.equals(s, "")).
-                collect(Collectors.toList())).hasSize(1);
+        assertThat(artifacts.stream()
+                .map(artifact -> artifact.askForGet(ArtifactSourceUrl.class))
+                .filter(Optional::isPresent)
+                .map(Optional::get)
+                .filter(s -> !"".equals(s))
+                .collect(Collectors.toList())).hasSize(1);
 
-        assertThat(artifacts.stream().
-                map(artifact -> artifact.askForGet(ArtifactSourceUrl.class)).
-                filter(Optional::isPresent).
-                map(Optional::get).
-                filter(s -> !Objects.equals(s, "")).
-                collect(Collectors.toList())).contains("https:/some.jar");
+        assertThat(artifacts.stream()
+                .map(artifact -> artifact.askForGet(ArtifactSourceUrl.class))
+                .filter(Optional::isPresent)
+                .map(Optional::get)
+                .filter(s -> !Objects.equals(s, ""))
```

Comment:
for consistency:
```suggestion
                .filter(s -> !Objects.equals(s, ""))
                .filter(s -> !"".equals(s))
```

Reply:
Since this are 6 lines of mostly duplicated code it might be good to extract the variable and compute it only once, e.g.
```java
    List<String> sourceUrls = artifacts.stream().map(...)...
```